src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -165,6 +167,30 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
 
             try
             {
+                // The current thread will be the event raising thread if either:
```

Comment:
👍 I'll change it to "event-raising" thread so the reader naturally associates it with `eventRaisingThreadId`.